### ISSUE TITLE
HTML5 Fish Bowl: None of the controls are keyboard accessible, & don't have labels

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#181]
Try to reach the fish drop down or the layers checkboxes on https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/fishbowlie/

Expected:

Work like native controls

Actual:

Custom controls that are not reachable or envokable via the keyboard (probably have no a11y roles etc either)

Recommendation:

Replace with native controls.

**Comment by molant** [comment]
Don't think we should change the code of any of the old performance demos. They were write like that for a reason and are used as benchmarks in different places.

**Comment by dstorey** [comment]
This also fails the accessibility criteria about controls having labels. I'll reuse this issue as should be fixed at the same time.

**Comment by sarvaje** [comment]
@dstorey @molant is ok if we just add the tabindex property, or is it necessary to change to native controls?

**Comment by dstorey** [comment]
@sarvaje Providing invoking the fake buttons work then tab index is fine, but you also need to add role="button" or whatever control so the screen reader knows what control it is. Using real controls is probably easier unless that will require a lot of refactoring for the styles.
